### ISSUE TITLE
crash:  invalid memory address or nil pointer dereference [2]

### ISSUE THREAD

**Issue by degeri** [decred/dcrpool#118]
If I click on the "backup" option in the admin panel the pool is crashing 

```
2019/07/26 xx:16:21 http2: panic serving 127.0.0.1:47428: runtime error: invalid memory address or nil pointer dereference
goroutine 314 [running]:
net/http.(*http2serverConn).runHandler.func1(0xc0003300c0, 0xc00062bfaf, 0xc0001e4c00)
        /usr/local/go/src/net/http/h2_bundle.go:5651 +0x16b
panic(0xaf1cc0, 0x134d420)
        /usr/local/go/src/runtime/panic.go:522 +0x1b5
github.com/coreos/bbolt.(*DB).beginTx(0x0, 0xaa0c40, 0xdf0a20, 0xc0002612a0)
        /home/r00t/go/pkg/mod/github.com/coreos/bbolt@v1.3.3/db.go:551 +0x26
github.com/coreos/bbolt.(*DB).Begin(0x0, 0xc00062bb00, 0xc0000b86c8, 0xba4573bc90b501, 0x500000000000007)
        /home/r00t/go/pkg/mod/github.com/coreos/bbolt@v1.3.3/db.go:544 +0x6e
github.com/coreos/bbolt.(*DB).View(0x0, 0xc00062bb18, 0x0, 0x0)
        /home/r00t/go/pkg/mod/github.com/coreos/bbolt@v1.3.3/db.go:709 +0x3c
github.com/decred/dcrpool/gui.(*GUI).PostBackup(0xc00008e580, 0xe0f7a0, 0xc0003300c0, 0xc00022b900)
        /home/r00t/workdir/dcrpool/gui/admin.go:137 +0x13f
net/http.HandlerFunc.ServeHTTP(0xc000123670, 0xe0f7a0, 0xc0003300c0, 0xc00022b900)
        /usr/local/go/src/net/http/server.go:1995 +0x44
github.com/gorilla/csrf.(*csrf).ServeHTTP(0xc000434820, 0xe0f7a0, 0xc0003300c0, 0xc00022b800)
        /home/r00t/go/pkg/mod/github.com/gorilla/csrf@v1.5.1/csrf.go:267 +0x46f
github.com/gorilla/mux.(*Router).ServeHTTP(0xc00013c0c0, 0xe0f7a0, 0xc0003300c0, 0xc000668100)
        /home/r00t/go/pkg/mod/github.com/gorilla/mux@v1.7.2/mux.go:212 +0xe3
net/http.serverHandler.ServeHTTP(0xc0002a2000, 0xe0f7a0, 0xc0003300c0, 0xc000668100)
        /usr/local/go/src/net/http/server.go:2774 +0xa8
net/http.initNPNRequest.ServeHTTP(0xc00014c700, 0xc0002a2000, 0xe0f7a0, 0xc0003300c0, 0xc000668100)
        /usr/local/go/src/net/http/server.go:3323 +0x8d
net/http.(*http2serverConn).runHandler(0xc0001e4c00, 0xc0003300c0, 0xc000668100, 0xc000662780)
        /usr/local/go/src/net/http/h2_bundle.go:5658 +0x89
created by net/http.(*http2serverConn).processHeaders
        /usr/local/go/src/net/http/h2_bundle.go:5392 +0x4f4
```

**Comment by degeri** [comment]
Closed by #119

**Comment by dnldd** [comment]
Fixed by #119.
